shared/wallets/wallet-list/container.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import {WalletList} from '.'
+import logger from '../../logger'
+import {connect, type TypedState, type Dispatch} from '../../util/container'
+import {getAccounts} from '../../constants/wallets'
+
+const mapStateToProps = (state: TypedState) => ({accounts: getAccounts(state)})
+
+const mapDispatchToProps = (dispatch: Dispatch, {navigateUp}) => ({
+  onAddNew: () => {
+    logger.error('TODO: onAddNew')
+  },
+  onLinkExisting: () => {
+    logger.error('TODO: onLinkExisting')
+  },
+})
+
+const mergeProps = (stateProps, dispatchProps) => {
+  // Accounts is already ordered, so no need to sort.
+  const wallets = stateProps.accounts
+    .map(a => {
+      let name = a.name || a.accountID
+      // TODO: Better way to do this?
```

Comment:
Do we have something that already does this? If the wallet has no name, then the full account ID is too long.

Reply:
this should likely just be in teh wallet object as the name